bigtable/client/version.h

```diff
@@ -15,9 +15,16 @@
 #ifndef GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_VERSION_H_
 #define GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_VERSION_H_
 
+#ifndef _MSC_VER
+// Microsoft Vistual Studio does not define __cplusplus correctly for C++11.
 #if __cplusplus < 201103L
 #error "Bigtable C++ Client requires C++11"
 #endif  // __cplusplus < 201103L
+#elif _MSC_VER < 1700
```

Comment:
If I am not wrong default and delete functions are not supported by _MSC_VER 1700 according to [this](https://msdn.microsoft.com/en-gb/library/hh567368.aspx). And I think we need this feature.

Reply:
Fixed.  Check for 1900.